docs/examples/01-exploring-wellcome-collections-apis.md

```diff
@@ -1,6 +1,6 @@
 # 1. Exploring Wellcome Collection's APIs
 
-[View on GitHub](https://github.com/wellcomecollection/developers.wellcomecollection.org/blob/main/notebooks/01-exploring-wellcome-collections-apis.ipynb) | [Run in Google Colab](https://colab.research.google.com/github/wellcomecollection/developers.wellcomecollection.org/blob/main/notebooks/01-exploring-wellcome-collections-apis.ipynb)
+[View on GitHub](https://github.com/wellcomecollection/developers.wellcomecollection.org/blob/add-events/notebooks/01-exploring-wellcome-collections-apis.ipynb) | [Run in Google Colab](https://colab.research.google.com/github/wellcomecollection/developers.wellcomecollection.org/blob/add-events/notebooks/01-exploring-wellcome-collections-apis.ipynb)
```

Comment:
I don't get why these have all been changed from `main` to `add-events`?

Reply:
This repo always makes changes in its PRs to change the internal links to match the branch, so you can navigate within it. They change it back when you merge. I haven't touched those files!